virtualizarr/manifests/manifest.py

```diff
@@ -57,7 +57,7 @@ def to_kerchunk(self) -> tuple[str, int, int]:
         """Write out in the format that kerchunk uses for chunk entries."""
         return (self.path, self.offset, self.length)
 
-    def dict(self) -> ChunkDictEntry:
+    def dict(self) -> ChunkDictEntry:  # type: ignore
```

Comment:
I... don't know why I ignored this. I'll double check.

Reply:
Ah, the `dict()` method is incompatible with the base class. I've tightened up the ignore, but will remove it entirely in #210